### ISSUE TITLE
[Keyboard navigation-Help-JavaScript-call a function]: Visual focus indicator is not visible on ‘Side documentation’ button on call a function pane.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2534]
**User Experience:**
Users who depends on Keyboard will feel difficulty to access any controls if Focus indicator is not visible on that.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Micro bit section element and select Code control given on it.
3-Navigate to the Help control lying in the header section on the page and select it.
4-Navigate to 'JavaScript' control and select it.
5-Navigate to various controls lying on the pane opened.
6-Navigate to the pane opened and select any link listed under it.
7-Verify visual focus indicator is visible on ‘Side documentation’ button or not.

**Actual Result**
Visual focus indicator is not visible on ‘Side documentation’ button on call a function pane. After zoom out button focus moving to side documentation button and side documentation button is collapsing and expanding but visual focus indicator is not visible on side documentation button. 

**Expected Result**
Visual focus indicator should be visible on ‘Side documentation’ button on call a function pane.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Suggested fix:** 
1.Using user interface components that are highlighted by the user agent when they receive focus 
2.Using CSS to change the presentation of a user interface component when it receives focus  
3.Using the default focus indicator for the platform so that high visibility default focus indicators will carry over  
4.Using an author-supplied, highly visible focus indicator  
5.Using script to change the background color or border of the element with focus   
#div a:hover, #div a:active, #div a:focus { 
  background-color: #DCFFFF; 
  color:#000066; 
} 
To provide an outline around the element we can use the below  
a { outline: none; }  

**Please Refer attachment for more details**
[MAS2.4.7_Call a function_Visual focus indicator is not visible on siade a documentation button.zip](https://github.com/Microsoft/pxt/files/1179073/MAS2.4.7_Call.a.function_Visual.focus.indicator.is.not.visible.on.siade.a.documentation.button.zip)

**Comment by v-amik** [comment]
review done

**Comment by v-mesala** [comment]
Verified on URL: and issue is not fixed. Keyboard dotted line focus indicator is not visible on side documentation button. Hence reactivating the bug.

**Comment by veler** [comment]
The dotted line focus indicator is proper to Edge. Let's discuss in our next UI meeting whether a background color change is not enough and if a outline is necessary.

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/acc and issue is fixed now. Hence closing the bug.
